int32 pidx
int32 oidx
---
bool dec
float64 conf
